app/src/main/java/ch/epfl/qedit/view/home/HomeQuizListFragment.java

```diff
@@ -21,39 +27,88 @@
 public class HomeQuizListFragment extends Fragment {
     public static final String QUIZID = "ch.epfl.qedit.view.QUIZID";
```

Comment:
Why do we need this id here?

Reply:
You wrote it you tell me

Reply:
oh, i did not write this part :) i will ask Nicolas

Reply:
This was useful before we removed the method loadquiz in the QuizViewModel. Via the extra I passed the id from the HomeQuizListFragment further down.